cogent/COGENT/Desugar.hs

```diff
@@ -370,7 +370,10 @@ desugarType t = typeWHNF t >>= \case
   S.RT (S.TUnit)   -> return TUnit
   notInWHNF -> __impossible $ "desugarType" ++ show notInWHNF
 
+-- What is this even doing here?
```

Comment:
@liamoc same as the whnf function in typechecker. typechecker doesn't convert types to whnf in the output.